### ISSUE TITLE
I think black background better

### ISSUE THREAD

**Issue by studiotang** [DennisSnijder/MakeGithubGreatAgain#47]
See title

**Comment by welkie** [comment]
I agree. It bothered me for a few days but then I realized the black only appears when you're logged in. It makes a lot of sense now.

**Comment by adriaanvanrossum** [comment]
> I realized the black only appears when you're logged in. (...)

It (now) has a dark background when you're not logged in:

![image](https://user-images.githubusercontent.com/1079135/32517615-f0f70aea-c3d4-11e7-8df3-8ad9630c0f23.png)

It doesn't bother me anyhow.
